tutorials/index.md

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
Did you meant to replace this file with `public-health-ug/index.md`?

Reply:
Yes - I thought that addressed the comment about splitting PRs/creating separate tutorials? As you can see I am new to this and am finding the process rather confusing.. sorry